powerfulseal/policy/action_probe_http.py

```diff
@@ -78,25 +78,28 @@ def make_call(self, url, method, body, headers, timeout, code, proxy, verify):
             "Making a call: %s, %s, %r, %d, %d, %s, %s, %s",
             url, method, headers, timeout, code, body, proxy, verify
         )
-        try:
-            resp = requests.request(
-                method.upper(),
-                url,
-                headers=headers,
-                timeout=timeout/1000,
-                data=body.encode("utf-8"),
-                proxies=dict(
-                    http=proxy or "",
-                    https=proxy or "",
-                ),
-                verify=verify,
+        resp = requests.request(
```

Comment:
Should just the call to `requests.request()` be wrapped with a try? Log an error and return False if `requests.request()` itself raises an exception.

Reply:
Yeah, probably a good idea. Will do.